js/ui/date_box/ui.date_box.strategy.calendar.js

```diff
@@ -147,6 +151,7 @@ var CalendarStrategy = DateBoxStrategy.inherit({
         if(dateBox.option("applyValueMode") === "instantly") {
             dateBox.option("opened", false);
             this.dateBoxValue(this.getValue(), e.event);
+            this._lastActionElement = null;
```

Comment:
Should we reset the `_lastActionElement` when dropDown closed?

Reply:
You are right, it's not necessary cause this variable will be set to "calendar" or "input" after any action while dropDown is opened